### ISSUE TITLE
Turn the redirect page into an app landing page

### ISSUE THREAD

**Issue by cassidyjames** [elementary/appstream-redirect#10]
@btkostner says the hard part is getting up-to-date appstream data. We'll have to figure that out before we can actually do this.

**Comment by danirabbit** [comment]
Duplicate of #9 don't you think? :p

**Comment by cassidyjames** [comment]
I think #9 is a step in this direction, but that an app landing page is separate and distinct from a generic non-redirect page. Could be solved in the same way, but #9 could also be solved first while this would be more work.

**Comment by cassidyjames** [comment]
So the "solution" I've been using over at [Appage](https://cassidyjames.github.io/appage/) is a Ruby script that hits the Appstream servers, downloads all the appropriate stuff, and regenerates the static site from that data. And it's actually worked pretty well, minus the obvious delay in time from publishing to when the script is run. But if it was just a daily cron job or something, I honestly think that would be enough.

Bumping this issue because I'd really love to get these combined, and it's something users have actively been asking for. A web based browsble store is huge for marketing and outreach (and a bit of FOMO).
